whelk-core/src/main/groovy/whelk/Document.groovy

```diff
@@ -508,26 +508,24 @@ class Document {
         return null
     }
 
+    Set<String> getEmbellishmentsIris() {
+        Set<String> result = new HashSet<>()
+        data[JsonLd.GRAPH_KEY].eachWithIndex{ def entry, int i ->
+            if (i > 1 && entry[JsonLd.GRAPH_KEY]) {
```

Comment:
Maybe `data[JsonLd.GRAPH_KEY][2..-1]` instead of `.eachWithIndex` plus `i > 1` check? (Or a streaming equivalent, since why not also findAll+collect on entries with graph keys, etc.) Mainly nitpicking...

Reply:
:+1: